### PR TITLE
Topic2

### PR DESCRIPTION
**Pull request by jaiup**

### CODE REVIEW

File: 02-oop-inheritance/js/index.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+class EventEmitter {
+  constructor() {
+    this.events = [];
+  }
+
+  on(event, listener) {
+    let eventFound = this.events.find(function(element) {
+      if (event === element.name) return element;
```
Comment (afiorenza) [inline 02-oop-inheritance/js/index.js]: This is okey, but for reading and maintainability purpose the if block should be 

```
if (condition) {
  statements
}
```

event if the statement is one line

File: 02-oop-inheritance/js/index.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  }
+}
+
+class Logger {
+  constructor() {
+
+  }
+
+  log(info) {
+    console.log("The '" + info + "' event has been emmited");
```
Comment (afiorenza) [inline 02-oop-inheritance/js/index.js]: Can you pass the movie and log the title? Also use template string. Thanks!

**Review by afiorenza — CHANGES_REQUESTED** [review]
Perfect work! just a few changes!

File: 02-oop-inheritance/js/index.js (hunk 1 above)
Comment (jaiup) [reply 02-oop-inheritance/js/index.js]: ok

File: 02-oop-inheritance/js/index.js (hunk 2 above)
Comment (jaiup) [reply 02-oop-inheritance/js/index.js]: ok

**Review by afiorenza — APPROVED** [review]
Good job! Ready for merge.Congrats!

**Pull request merged**